lib/prompts/downloadProjectPrompt.ts

```diff
@@ -1,36 +1,53 @@
-// @ts-nocheck
-const { promptUser } = require('./promptUtils');
-const { getAccountId } = require('@hubspot/local-dev-lib/config');
-const { fetchProjects } = require('@hubspot/local-dev-lib/api/projects');
-const { logError, ApiErrorContext } = require('../errorHandlers/index');
-const { EXIT_CODES } = require('../enums/exitCodes');
-const { i18n } = require('../lang');
+import { promptUser } from './promptUtils';
+import { getAccountId } from '@hubspot/local-dev-lib/config';
+import { fetchProjects } from '@hubspot/local-dev-lib/api/projects';
+import { logError, ApiErrorContext } from '../errorHandlers/index';
+import { EXIT_CODES } from '../enums/exitCodes';
+import { i18n } from '../lang';
+import { Project } from '@hubspot/local-dev-lib/types/Project';
 
 const i18nKey = 'lib.prompts.downloadProjectPrompt';
 
-const createProjectsList = async accountId => {
+type DownloadProjectPromptResponse = {
+  project: string;
+};
+
+async function createProjectsList(
+  accountId: number | null
+): Promise<Project[]> {
   try {
-    const { data: projects } = await fetchProjects(accountId);
-    return projects.results;
+    if (accountId) {
+      const { data: projects } = await fetchProjects(accountId);
+      return projects.results;
+    }
+    return [];
```

Comment:
It looks like this may have subtly changed the behavior of this function - previously if there was no `accountId`, an error would have been thrown but now this return an empty array. This also also means the `else` block within the `catch` will never actually happen. This is maybe OK but i'd either double check that this doesn't change the behavior of the prompt or just throw an error if there's no `accountId` present

Reply:
Decided to throw an error if `accountId` isn't present.

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1288/commits/d73f2185f196a582671543443e00908845022b7c